tests/trace/test_annotation_feedback.py

```diff
@@ -456,3 +456,60 @@ class FeedbackModel(BaseModel):
     # Invalid cases should return False
     assert not enum_spec.value_is_valid("invalid_choice")
     assert not enum_spec.value_is_valid(123)
+
+
+def test_annotation_feedback_sdk(client):
```

Comment:
leftover?

Reply:
yeah oops that must have happened last night, branch cross pollination

Reply:
or rather, this is pointing at an outdated head I think. This code is not unique to this branch